packages/migrate/commonsChunkPlugin/__snapshots__/commonsChunkPlugin.test.js.snap

```diff
@@ -0,0 +1,295 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-0" data 1`] = `
+"module.exports = {
+    entry: {
+		app: './src/app.js',
+		vendor: './src/vendors.js',
+	},
+
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                app: {
+                    name: 'app',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 2,
+                    test: 'app'
+                },
+
+                vendor: {
+                    name: 'vendor',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 2,
+                    test: 'vendor'
+                }
+            }
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-1" data 1`] = `
+"module.exports = {
+    entry: {
+		vendor: './src/vendors.js',
+	},
+
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                common: {
+                    name: 'common',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 2
+                },
+
+                vendor: {
+                    name: 'vendor',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 2,
+                    test: 'vendor'
+                }
+            }
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-2" data 1`] = `
+"module.exports = {
+    entry: {
+		vendor: './src/vendors.js',
+	},
+
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                vendor: {
+                    name: 'vendor',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 1,
+                    test: 'vendor'
+                }
+            }
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-3" data 1`] = `
+"module.exports = {
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                commons: {
+                    name: 'commons',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 1,
+                    filename: 'commons.js'
+                }
+            }
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-4" data 1`] = `
+"module.exports = {
+    entry: {
+		main: './src/index.js',
+	},
+
+    optimizations: {
+        splitChunks: {
+            chunks: 'async',
+            minSize: 2000,
```

Comment:
I would move this into the `cacheGroups.main`

Reply:
Still not resolved.

This case is not 100% map-able to splitChunks. This might be the best option:

``` js
            cacheGroups: {
                main: {
                    minSize: 2000,
                    chunks: 'async'
                }
            }
```